src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/Services/HttpClientWrapper.cs

```diff
@@ -7,23 +7,31 @@
 using System.Threading.Tasks;
 using Newtonsoft.Json;
 using SFA.DAS.EAS.Domain.Http;
+using SFA.DAS.EAS.Domain.Interfaces;
 using SFA.DAS.NLog.Logger;
 
 namespace SFA.DAS.EAS.Infrastructure.Services
-{
+{ 
     public class HttpClientWrapper : IHttpClientWrapper
     {
         public string AuthScheme { get; set; }
         public string BaseUrl { get; set; }
         public List<MediaTypeWithQualityHeaderValue> MediaTypeWithQualityHeaderValueList { get; set; }
         private readonly ILog _logger;
 
+        private readonly IHttpResponseLogger _httpResponseLogger;
+
         public HttpClientWrapper(ILog logger)

```

Comment:
If there are multiple constructors on a class then StructureMap's default behavior is to select the greediest, so I think we can remove this? Will it blow up without an `IHttpResponseLogger` if a 400 response is returned anyway?

Reply:
Cheers.
Are rats! Good spot - I put this additional constructor to in there while I was trying to figure out the ioc/multiple nlogger issue and it stuck in.